### PR TITLE
remove duplicate assert

### PR DESCRIPTION
**Pull request by yuhongliang90**
duplicate assert

### CODE REVIEW

File: src/layer/GeoJSON.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ export function geometryToLayer(geojson, options) {
 function _pointToLayer(pointToLayerFn, geojson, latlng, options) {
 	return pointToLayerFn ?
 		pointToLayerFn(geojson, latlng) :
-		new Marker(latlng, options && options.markersInheritOptions && options);
+		new Marker(latlng, options && options.markersInheritOptions);
```
Comment (Falke-Design) [inline src/layer/GeoJSON.js:238]: the return value is now not `options` anymore, this needs more investigations

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: src/layer/GeoJSON.js (hunk 1 above)
Comment (IvanSanchez) [reply src/layer/GeoJSON.js:238]: The first `options &&` ensures that `options` is an non-undefined object, whereas the second `&& options` handles the semantics of `markersInheritOptions` (returning `options` if `markersInheritOptions` is truthy)

Should be `options?.markersInheritOptions && options` instead, or maybe `options?.markersInheritsOptions ? options : {}`. The behaviour is the same, but the fancy syntax makes the purpose a bit clearer IMO.

**Review by IvanSanchez — CHANGES_REQUESTED** [review]

File: src/layer/GeoJSON.js (hunk 1 above)
Comment (Falke-Design) [reply src/layer/GeoJSON.js:238]: I prefer `options?.markersInheritOptions && options`

**Comment by Falke-Design** [comment]
I'm closing this because everything works as designed

**Pull request closed without merging**